### ISSUE TITLE
Fix overflow in https://ocaml.org/learn/tutorials/99problems.html on smaller screens

### ISSUE THREAD

**Issue by TildaDares** [ocaml/v2.ocaml.org#1379]
Some of the code blocks in this [page](https://ocaml.org/learn/tutorials/99problems.html) are causing an overflow because the texts are not wrapping on smaller screen sizes.

![image](https://user-images.githubusercontent.com/63427719/113462755-6b963a00-941a-11eb-8248-d1283056d4a7.png)

I'd like to fix it by setting the white-space property to normal so that the texts can wrap and the overflow can be removed.

![image](https://user-images.githubusercontent.com/63427719/113462818-aac48b00-941a-11eb-9bd4-b875ba0e8d80.png)

**Comment by pitag-ha** [comment]
To me it sounds like a good idea to wrap inlined code as the one for which you've provided the screenshot! I wouldn't inline code blocks though (I'm not sure if you also had that in mind or not). You can go ahead and work on wrapping inlined code :) For consistency, please, make sure you do that on the whole page.

**Comment by TildaDares** [comment]
The code tag has a `white-space` property of `nowrap`. I was thinking of setting to `normal`.

**Comment by pitag-ha** [comment]
Is the code tag the tag used for inlined code (for example `hs = ...` in your screenshot)? If so, that sounds like a good improvement!

**Comment by TildaDares** [comment]
Yes but I'd have to replace the backticks with an actual code tag so that I can add a class. Is that okay?
